python/ray/ray_constants.py

```diff
@@ -60,13 +63,20 @@ def env_bool(key, default):
 # print a warning.
 DUPLICATE_REMOTE_FUNCTION_THRESHOLD = 100
 
-# The maximum resource quantity that is allowed. TODO(rkn): This could be
-# relaxed, but the current implementation of the node manager will be slower
-# for large resource quantities due to bookkeeping of specific resource IDs.
-MAX_RESOURCE_QUANTITY = 100000
+if _config.new_scheduler_enabled:
+    MAX_RESOURCE_QUANTITY = 2**63 / 10**4
+else:
+    # The maximum resource quantity that is allowed. TODO(rkn): This could be
+    # relaxed, but the current implementation of the node manager will be
+    # slower for large resource quantities due to bookkeeping of specific
+    # resource IDs.
+    MAX_RESOURCE_QUANTITY = 100000
 
 # Each memory "resource" counts as this many bytes of memory.
-MEMORY_RESOURCE_UNIT_BYTES = 50 * 1024 * 1024
+if _config.new_scheduler_enabled:
+    MEMORY_RESOURCE_UNIT_BYTES = 1
```

Comment:
Shouldn't this be 1024 * 1024 = 1 MiB?

Reply:
Whoops I forgot to update the description, but bytes seems a little nicer than MiB?